### PR TITLE
Update OpenAPI specification

### PR DESCRIPTION
**Pull request by androa**

### CODE REVIEW

File: openapi-definition.yaml [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+components:
+  schemas:
+
+    Subsumsjon:
+      type: object
+      required:
+        - beregningsId
+        - opprettet
+        - utført
```
Comment (nicokk) [inline openapi-definition.yaml]: I see you have avoided using æøå in other names, should this be changed to 'utfort'?

File: openapi-definition.yaml [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+          type: boolean
+          default: false
+          description: Om bruker har hatt inntekt i EØS.
+        bruktInntektsPeriode:
+          $ref: '#/components/schemas/InntektsPeriode'
+
+    MinsteinntektResultat:
+      required:
+        - oppfyllerKravetTilMinsteArbeidsinntekt
+        - sisteMaaned
```
Comment (nicokk) [inline openapi-definition.yaml]: sisteMaaned or periodeAntallUker?

File: openapi-definition.yaml (hunk 1 above)
Comment (androa) [reply openapi-definition.yaml]: Good point 👍 

```suggestion
        - utfort
```

File: openapi-definition.yaml (hunk 2 above)
Comment (androa) [reply openapi-definition.yaml]:
```suggestion
        - periodeAntallUker
```

**Pull request merged**